src/test/java/com/atlassian/db/replica/it/example/aurora/AuroraClusterTest.java

```diff
@@ -61,22 +65,22 @@ public void shouldUtilizeReplicaForReadQueriesForSynchronisedWrites() throws SQL
     private SqlCall<Connection> initializeConnectionPool(final DatabaseCall decisionLog) throws SQLException {
         final ReplicaNode replicaNode = new ReplicaNode();
         final ConnectionProvider connectionProvider = new AuroraConnectionProvider(
```

Comment:
Should `AuroraConnectionProvider` accept Strings?

Reply:
I gotta rethink ConnectionProvider, I wonder if it should be more multi-replica friendly